src/screens/ForgottenPasswordScreen.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { View, Dimensions } from 'react-native'
+import { LinearGradient } from 'expo-linear-gradient'
+import AuthHeader from '@/components/headers/AuthHeader'
+import Step1 from '@/components/forgotPassword/Step1'
+import Step2 from '@/components/forgotPassword/Step2'
+import Step3 from '@/components/forgotPassword/Step3'
+import Step4 from '@/components/forgotPassword/Step4'
+import { SetupProps } from '@/types/props'
+
+type Props = SetupProps<'ForgotPassword'>
+
+function ForgottenPasswordScreen(props: Props) {
+  const { navigation } = props
+
+  const windowHeight = Dimensions.get('window').height
+
+  const [emailAddress, setEmailAddress] = React.useState('')
+  const [newPassword, setNewPassword] = React.useState('')
+  const [confirmPassword, setConfirmPassword] = React.useState('')
+  const [step, setStep] = React.useState(0)
+
+  return (
+    <LinearGradient
+      colors={['#ae60cc', '#7E5BFF']}
+      locations={[0, 0.6]}
+      className="w-full h-full px-5 pt-20 pb-12"
+      style={{ flex: 1 }}
+    >
+      <View className="flex flex-col justify-between h-full items">
+        <AuthHeader />
+
+        <View className="flex flex-col items-center justify-center"></View>
+
+        {step === 0 ? (
```

Comment:
If we go with the renaming of steps. We can probably do it via an enum and switch?

``` typescript
switch (step) {
    case Steps.SEND_EMAIL:
      return StepSendEmail // component
}
```

Reply:
👍